governator-core/src/main/java/com/netflix/governator/LifecycleInjector.java

```diff
@@ -102,4 +102,9 @@ public void shutdown() {
     public void addListener(LifecycleListener listener) {
         manager.addListener(listener);
     }
+    
+    @Override
+    public void close() {
+    	shutdown();
```

Comment:
we can deprecate it, but I'd definitely leave shutdown() present to avoid a breaking change